public/toolkit/styles/organisms/_property-card.scss

```diff
@@ -74,44 +74,58 @@
   }
 }
 
-.property-card_figure {
-  text-align: center;
-  position: relative;
-  line-height: .8;
+.property-card_figure-wrapper {
 
-  // Image with rounded corners
-  &.radius {
-    @media #{$medium-up} {
-      border-radius: $form-card-radius;
-    }
+  .property-card_figure {
+    text-align: center;
+    position: relative;
+    line-height: .8;
 
-    img,
-    .property-card_overlay:before {
+    // Image with rounded corners
+    &.radius {
       @media #{$medium-up} {
         border-radius: $form-card-radius;
       }
+
+      img,
+      .property-card_overlay:before {
+        @media #{$medium-up} {
+          border-radius: $form-card-radius;
+        }
+      }
+    }
+
+    @media #{$medium-up} {
+      background-color: $charcoal;
+      text-align: left;
     }
   }
 
-  @media #{$medium-up} {
-    background-color: $charcoal;
+  .badge-label {
+    position: absolute;
+    top: 1rem;
+    left: 50%;
+    transform: translateX(-50%);
+    z-index: 2;
+
+    @media #{$medium-up} {
+      left: 1rem;
+      transform: none;
+    }
+
   }
 
-  // Labels overlaying images
```

Comment:
curious what the motivation was for removing a couple of comments here.

Reply:
the labels referenced by the comment are no longer overlaying the image, so the comment didn't make any sense anymore. i can re-add one that _does_ make sense, but i think anyone looking at the scss + the markup should be able to intuit what's going on sans comments. your call!